packages/lib/src/utils/ABIs.ts

```diff
@@ -137,5 +146,6 @@ export function callDescription(method: any, args: string[]): string {
 export default {
   buildCallData,
   getABIFunction,
+  getABIType: getArgTypeLabel,
```

Comment:
Can't we remove this here? It didn't exist before this PR so there are no backwards compatibility issues.

Reply:
Yes, that was the auto-rename backfiring - sorry about it